src/main/java/models/Game.java

```diff
@@ -25,10 +25,18 @@ public Game(){
 
     public void buildDeck() {
         for(int i = 2; i < 15; i++){
-            deck.add(new Card(i,Suit.Clubs));
             deck.add(new Card(i,Suit.Hearts));
-            deck.add(new Card(i,Suit.Diamonds));
             deck.add(new Card(i,Suit.Spades));
+            deck.add(new Card(i,Suit.Diamonds));
+            deck.add(new Card(i,Suit.Clubs));
+        }
+    }
+    public void buildSDeck() {
+        for(int i = 2; i < 12; i++){
+            deck.add(new Card(i,Suit.Coins));
+            deck.add(new Card(i,Suit.Cups));
+            deck.add(new Card(i,Suit.Swords));
+            deck.add(new Card(i,Suit.Clubs));
         }
```

Comment:
ultimately these should be coming from a different deck class, so this will need to be changed when the abstract and child classes are made to follow the proposed UML design.